Fix build on OS X 10.7 and earlier
/usr/include/net/if.h:300:19: error: field has incomplete type 'struct sockaddr'
--- a/xdm/auth.c.orig	2011-09-25 02:35:47.000000000 -0500
+++ b/xdm/auth.c	2017-10-08 00:27:05.000000000 -0500
@@ -81,6 +81,7 @@
 # include <netdb.h>
 # undef SIOCGIFCONF
 #else /* __GNU__ */
+# include <sys/socket.h>
 # include <net/if.h>
 #endif /* __GNU__ */
 
